cpp/src/io/comp/decompression.cpp

```diff
@@ -109,11 +110,6 @@ struct zip_lfh_s {
   uint16_t extra_len;    // extra field length
 };
 
-struct bz2_file_header_s {
```

Comment:
Maybe I missed it but where has this struct been moved to? Or was it not in use and removed altogether?